Examples/Examples/WorldScaleExampleView.swift

```diff
@@ -52,7 +52,7 @@ struct WorldScaleGeoTrackingExampleView: View {
                 }
         }
         .calibrationViewHidden(false)
-        .calibrationViewAlignment(.bottomLeading)
+        .calibrationViewAlignment(.bottom)
```

Comment:
```suggestion
```
The alignment is set to `.bottom` by default.

Reply:
Maybe we should include it in the example so someone will be able to find the modifier easier? I'll revert my change. 🤔